lib/paperclip/io_adapters/identity_adapter.rb

```diff
@@ -1,8 +1,11 @@
 module Paperclip
   class IdentityAdapter < AbstractAdapter
-    def new(adapter)
+    def new(adapter, _)
       adapter
     end
+
+    def initialize
```

Comment:
I'll have to look at this closer later because at first glance I don't get what this class does. Changing it consistently breaks "Paperclip::Storage::S3 An attachment that uses S3 for storage and has a question mark in file name returns a replaced version for {path|url}" specs.